storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -326,16 +331,29 @@
                                          (log-error t "Error when processing event")
                                          (exit-process! 20 "Error when processing an event")
                                          ))
+   :blob-update-timer (mk-timer :kill-fn (fn [t]
+                                           (log-error t "Error when processing blob-update")
+                                           (exit-process! 20 "Error when processing a blob-update")
```

Comment:
it would make it more clear but i guess in order to keep it consistent with other's we might want to change it